_posts/0000-01-02-rezamalik.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"When I wrote this code, only God and I understood what I did. Now only God knows."